app/views/audits/_form.html.erb

```diff
@@ -22,7 +22,7 @@
                     </div>
                     <div class="row links">
                       <div class="col-xs-12 p-3">
-                        <%= add_element_button "Add Another Item", container_selector: "#audit_line_items" do %>
+                        <%= add_element_button "Add Another Item", container_selector: "#audit_line_items", id: "__add_line_item" do %>
```

Comment:
Noticed that this method call was also missing the `id` option.

I added some corresponding system specs, but I can remove them if you think the coverage isn't worth the extra weight of system specs.